x/babylon/contract/out_message.go

```diff
@@ -1,7 +1,16 @@
 package contract
 
+import "time"
+
 // SudoMsg is a message sent from the Babylon module to a smart contract
-// TODO: implement
 type SudoMsg struct {
-	TestSudoMsg *struct{} `json:"test_sudo_msg,omitempty"`
+	BeginBlockMsg *BeginBlock `json:"begin_block,omitempty"`
+}
+
+type BeginBlock struct {
+	Height     int64     `json:"height"`       // Height returns the height of the block
```

Comment:
```suggestion
	Height     int64     `json:"height"`       // Height is the height of the block
```
?

(The same with the other fields)